gadgets/bpfstats/README.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: bpfstats
+sidebar_position: 0
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# bpfstats
```

Comment:
I think we should mention the mechanism we are using to enable stats: BPF_ENABLE_STATS and fallback to `/proc/sys/kernel/bpf_stats_enabled`. 

In addition, we should highlight that for kernels < 5.8, where `BPF_ENABLE_STATS` is not available, we are not setting `kernel.bpf_stats_enabled` back to `0`.

Reply:
> In addition, we should highlight that for kernels < 5.8, where BPF_ENABLE_STATS is not available, we are not setting kernel.bpf_stats_enabled back to 0.
>
And probably create an issue for this.

Reply:
#4319